manim/cli/render/commands.py

```diff
@@ -71,18 +69,32 @@ def render(
         logger.warning(
             "--use_opengl_renderer is deprecated, please use --renderer=opengl instead!"
         )
-        renderer = "opengl"
+        args["renderer"] = "opengl"
 
     if args["use_webgl_renderer"]:
         logger.warning(
             "--use_webgl_renderer is deprecated, please use --renderer=webgl instead!"
         )
-        renderer = "webgl"
+        args["renderer"] = "webgl"
 
     if args["use_webgl_renderer"] and args["use_opengl_renderer"]:
         logger.warning("You may select only one renderer!")
         sys.exit()
 
+    if args["save_as_gif"]:
+        logger.warning("--save_as_gif is deprecated, please use --format=gif instead!")
+        args["format"] = "gif"
+
+    if args["save_pngs"]:
+        logger.warning("--save_pngs is deprecated, please use --format=pngs instead!")
+        args["format"] = "pngs"
+
+    if args["save_last_frame"]:
+        logger.warning(
+            "--save_last_frame is deprecated, please use --format=png instead!"
```

Comment:
`png` vs `pngs` is sort of confusing. What I would suggest is keeping `save_last_frame` for saving the last frame and `--format=png` would go for saving all the frames as png. If both `--format=png` and `save_last_frame` is passed it would be better to just prefer `save_last_frame` and warn about  `--format=png`.